.gitignore

```diff
@@ -71,3 +71,6 @@ pip-delete-this-directory.txt
 
 # pytest report
 ui-test.html
+pytest-selenium/tests/test_attribution_pages.py
+pytest-selenium/tests/test_attribution_pom.py
+.idea/
```

Comment:
please use your [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) for editor files

Reply:
@TomWoodward I removed these from the .gitignore and committed the changes.